### PR TITLE
Make npm install step optional in CLI wizard

### PR DESCRIPTION
**Pull request by gpt14**
## Proposed changes

Make npm install step optional in CLI wizarrd as detailed in #7864 

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md) doc
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)

## Further comments
NA

### CODE REVIEW

**Comment by gpt14** [comment]
@christian-bromann Need your help fixing the 2 failing tests

**Comment by christian-bromann** [comment]
@gpt14 how can I help you?

**Comment by christian-bromann** [comment]
@gpt14 please rebase on top of current master as I added some fixes to the build. Also let me know how we can move forward with this PR.

**Comment by gpt14** [comment]
Sure, @christian-bromann. 
I have made changes to the code which implements this functionality
https://github.com/gpt14/webdriverio/commit/2e27b5ead2bb97a464da81738f10fbf09e082dcd#diff-bbc05cc47b4181c7a97fb75f6094eb8a740b542733cb47e160ffc3a4508f5aaeR170

image

However, I am having trouble fixing these tests after my changes in the `config.test.js` file.
image

I was wondering if you might have any pointers to fix them?

**Comment by christian-bromann** [comment]
> I was wondering if you might have any pointers to fix them?

It seems like the `yarnInstall` mock is never been called. I recommend to run jest in watch mode:

```sh
$ npx jest ./packages/wdio-config/tests/commands/config.test.ts --watch --collectCoverage=false
```

And debug the test and config command function to find out what is going on.

**Review by christian-bromann — COMMENTED** [review]
Awesome, can we add a test that uses `npmInstall: false`?

**Review by christian-bromann — COMMENTED** [review]
I'ld suggest to add a `console.log` telling the user to install necessary dependencies to run the project. What do you think?

**Comment by gpt14** [comment]
Sounds good, @christian-bromann. 

Does this look ok to you?
image

File: packages/wdio-cli/src/constants.ts [hunk 1]
```diff
@@ -24,6 +24,13 @@ To run your tests, execute:
 $ npx wdio run %swdio.conf.%s
 `
 
+export const DEPENDENCIES_INSTALLATION_MESSAGE = `
+To install dependencies, execute \n
```
Comment (christian-bromann) [inline packages/wdio-cli/src/constants.ts:28]:
```suggestion
To install dependencies, execute:
```

**Review by christian-bromann — APPROVED** [review]
Awesome, thanks a ton 👍

**Pull request merged**